controllers/manifest_controller/helper/helper.go

```diff
@@ -21,11 +25,22 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/google/go-containerregistry/pkg/v1/types"
-	. "github.com/onsi/gomega"
+	gmg "github.com/onsi/gomega"
```

Comment:
Yes, the reason is it's not a `_test` package - it's the "normal" one - and the linter complains about dot imports. I could suppress the linter or use explicit package naming.

Reply:
Ok, got the point - but since this is anyway just for pure testing, lets suppress it I would say.
Please also check my other comment regards the file name